public/scripts/power-user.js

```diff
@@ -4072,4 +4072,34 @@ $(document).ready(() => {
         ],
         helpString: 'activates a movingUI preset by name',
     }));
+    SlashCommandParser.addCommandObject(SlashCommand.fromProps({
+        name: 'stop-strings',
+        aliases: ['stopping-strings'],
```

Comment:
```suggestion
        aliases: ['custom-stopping-strings'],
```
That'd be okay?
I was confused at first, typed it like it's named in the settings, and couldn't find anything starting to type "custom".

Reply:
Added a couple of aliases.